be/src/olap/rowset/rowset_meta.cpp

```diff
@@ -225,6 +225,7 @@ void RowsetMeta::merge_rowset_meta(const RowsetMeta& other) {
     set_data_disk_size(data_disk_size() + other.data_disk_size());
     set_total_disk_size(total_disk_size() + other.total_disk_size());
     set_index_disk_size(index_disk_size() + other.index_disk_size());
+    set_total_disk_size(data_disk_size() + index_disk_size());
```

Comment:
It is confusing to set total_disk_size twice.

Reply:
Yes, it needs just one set operation.